### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by lanre-ade**
- Allows toggling of image sharing on a per node basis
- Allows admin to approve newly registered users

### CODE REVIEW

**Comment by lanre-ade** [comment]
Not ready for review

**Comment by lanre-ade** [comment]
:wave:

File: api/admin/node.py [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+from django.contrib.admin import ModelAdmin, site
 
 class NodeOptions(ModelAdmin):
-    list_display = ['user', 'host', 'enabled', 'api_postfix', 'integrated', 'foreign_username']
+    list_display = ['user', 'host', 'enabled', 'api_postfix', 'integrated', 'image_sharing', 'foreign_username']
+
+    def share_images(self, request, queryset):
+        try:
+            queryset.update(image_sharing=True)
+            self.message_user(request, "successfully enabled image sharing")
```
Comment (derekdowling) [inline api/admin/node.py]: All of the message_user strings need to be cleaned up to "Uppercase sentence."

**Comment by derekdowling** [comment]
Other than cleaning up the user facing messages, this looks good. :+1:

**Pull request merged**